### PR TITLE
Dummydata 2023 10 05

### PR DESCRIPTION
**Pull request by epi-clz**
Hello Lore! This is for merging the dummy dataset to the main. Thanks

### CODE REVIEW

**Comment by loremerdrignac** [comment]
Dear @epi-gde,
I think that @epi-clz and I, we are happy with the current version of the code and we are ready for merging into the main.
I was wondering, are you happy that we upgrade the version number once this latest modification is included?
I see that you moved from v0.5.0.1 to v0.5.0.9-alpha but I am a bit lost with the figures :-/ sorry. Should we move to v0.5.1.0 then?
Thank you for your advice!

File: R/data.R [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#' 
+"RefSite"
+
+#' Dummy dataset
+#'
+#' Dataframe with commun epidemiological variables to use for tests
+#'
+#' @format A data frame with 100 observations on 8 variables.
+#' \describe{
+#'   \item{id}{Identifier}
```
Comment (loremerdrignac) [inline R/data.R:29]: Sorry Cristina I missed this! 
If we move to CamelCase, should we put ID, Country, Age and Case?
Let me know your thoughts! :)

**Comment by epi-gde** [comment]
> Dear @epi-gde, I think that @epi-clz and I, we are happy with the current version of the code and we are ready for merging into the main. I was wondering, are you happy that we upgrade the version number once this latest modification is included? I see that you moved from v0.5.0.1 to v0.5.0.9-alpha but I am a bit lost with the figures :-/ sorry. Should we move to v0.5.1.0 then? Thank you for your advice!

Yes alpha version are used for temporary version which are not ready for the final versioning ;)
You can now upgrade the version number and considering the huge amount of changes we can go for v0.5.1 
After merging SetType from Jenny and API from James , we may consider a V0.6.0  ;)

File: R/data.R (hunk 1 above)
Comment (epi-clz) [reply R/data.R:29]: i have not idea about this... maybe @epi-gde can confirm? But sure I can change it!

Comment (loremerdrignac) [reply R/data.R:29]: Hi Cristina! :)
I have upgraded the relase number to v0.5.1.0, as agreed with Gilles :)
Let me know what you decide for the column label! In any case, I would be in favor of using the same as with RefStie. But I am happy to change if you prefer the other spelling!

**Pull request merged**